src/doc/usage.md

```diff
@@ -35,7 +35,7 @@ A basic HTML5 Boilerplate site initially looks something like this:
 ├── doc
 ├── img
 ├── js
-│   ├── main.js
+│   ├── app.js
 │   ├── plugins.js
 │   └── vendor
 │       └── modernizr.min.js
```

Comment:
Normally in operating systems, code editors and/or _IDE_, folders are listed before files. In that case `vendor/` should be placed before `app.js`. Can we move folders before files on tree structure so it is listed properly?